phalcon/Mvc/Model/Query/Builder.zep

```diff
@@ -1687,4 +1681,20 @@ class Builder implements BuilderInterface, InjectionAwareInterface
 
         return this;
     }
+
+    /**
+     * Get first key from models array
+     */
+    private function getFirstModel(var models) -> string
```

Comment:
Do we really need a separated method for if/else which will be used once?

Reply:
Used in previous iteration for DRY approach. Decided to leave it here, to make `getPhql()` little bit lighter of code.

Reply:
And to make Phalcon slower I guess )